python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -1510,6 +1506,26 @@ def get_run():
         except DagsterRunAlreadyExists:
             return get_run()
 
+    def handle_started_run(self, run_record: RunRecord, logger: logging.Logger) -> None:
```

Comment:
I'm not super convinced this should be on the instance? The instance is already very out of control

Reply:
Originally had it on instance in case we wanted different behavior on different instance types, but I don't think it's actually needed.